var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -382,3 +382,23 @@ def check_install(self):
         spec = self.spec
         if "@master" in spec:
             self.smoke_test()
+
+    @property
+    def libs(self):
+        spec = self.spec
+        vsuffix = str(spec.version[:2])
+        libnames = [
+            "vtkm_rendering",
```

Comment:
- We are missing: `vtkmdiympi(_nompi)` and other potential libraries that are generated when certain options are enabled.
- We need this to be sorted